src/scripts/compute_impacted_targets.sh

```diff
@@ -30,49 +47,42 @@ fetchRemoteGitHistory() {
-fi
-
-if [[ -z ${WORKSPACE_PATH} ]]; then
-	echo "Missing workspace path"
-	exit 2
-fi
-
 fetchRemoteGitHistory "${MERGE_INSTANCE_BRANCH}"
 fetchRemoteGitHistory "${PR_BRANCH}"
 
-# Install the bazel-diff JAR. Avoid cloning the repo, as there will be conflicting WORKSPACES.
-curl --retry 5 -Lo bazel-diff.jar https://github.com/Tinder/bazel-diff/releases/latest/download/bazel-diff_deploy.jar
-java -jar bazel-diff.jar -V
-bazel --version
+## Verbose logging for the Merge Instance and PR branch.
+if [[ -n ${VERBOSE} ]]; then
+	git switch "${MERGE_INSTANCE_BRANCH}"
+	merge_instance_branch_head_sha=$(git rev-parse "${MERGE_INSTANCE_BRANCH}")
+	echo "Merge Instance Branch Head= ${merge_instance_branch_head_sha}"
 
-git switch "${MERGE_INSTANCE_BRANCH}"
-merge_instance_branch_head_sha=$(git rev-parse "${MERGE_INSTANCE_BRANCH}")
-logIfVerbose "Merge Instance Branch Head= ${merge_instance_branch_head_sha}"
+	git switch "${PR_BRANCH}"
+	pr_branch_head_sha=$(git rev-parse "${PR_BRANCH}")
+	echo "PR Branch Head= ${pr_branch_head_sha}"
 
-git switch "${PR_BRANCH}"
-pr_branch_head_sha=$(git rev-parse "${PR_BRANCH}")
-logIfVerbose "PR Branch Head= ${pr_branch_head_sha}"
+	# Find the merge base of the two branches
+	merge_base_sha=$(git merge-base "${merge_instance_branch_head_sha}" "${pr_branch_head_sha}")
+	echo "Merge Base= ${merge_base_sha}"
 
-# Find the merge base of the two branches
-merge_base_sha=$(git merge-base "${merge_instance_branch_head_sha}" "${pr_branch_head_sha}")
-logIfVerbose "Merge Base= ${merge_base_sha}"
+	# Find the number of commits between the merge base and the merge instance's HEAD
+	merge_instance_depth=$(git rev-list "${merge_base_sha}".."${merge_instance_branch_head_sha}" | wc -l)
+	echo "Merge Instance Depth= ${merge_instance_depth}"
 
-# Find the number of commits between the merge base and the merge instance's HEAD
-merge_instance_depth=$(git rev-list "${merge_base_sha}".."${merge_instance_branch_head_sha}" | wc -l)
-logIfVerbose "Merge Instance Depth= ${merge_instance_depth}"
+	git switch "${MERGE_INSTANCE_BRANCH}"
+	git log -n "${merge_instance_depth}" --oneline
 
-git switch "${MERGE_INSTANCE_BRANCH}"
-ifVerbose git log -n "${merge_instance_depth}" --oneline
+	# Find the number of commits between the merge base and the PR's HEAD
+	pr_depth=$(git rev-list "${merge_base_sha}".."${pr_branch_head_sha}" | wc -l)
+	echo "PR Depth= ${pr_depth}"
 
-# Find the number of commits between the merge base and the PR's HEAD
-pr_depth=$(git rev-list "${merge_base_sha}".."${pr_branch_head_sha}" | wc -l)
-logIfVerbose "PR Depth= ${pr_depth}"
+	git switch "${PR_BRANCH}"
+	git log -n "${pr_depth}" --oneline
+fi
 
-git switch "${PR_BRANCH}"
-ifVerbose git log -n "${pr_depth}" --oneline
+# Install the bazel-diff JAR. Avoid cloning the repo, as there will be conflicting WORKSPACES.
+curl --retry 5 -Lo bazel-diff.jar https://github.com/Tinder/bazel-diff/releases/latest/download/bazel-diff_deploy.jar
+java -jar bazel-diff.jar -V
+bazel --version
```

Comment:
This bazel call should also have the startup options, or else it will restart

Reply:
Oh, good catch, adding that.